File: Daily/July/22-July-2025/3.java
class Main {
    public static void main(String[] args) {
        int number = 15;
        // int fact = 1;
        for (int i=1; i<=number; i++) {
            if(number%i==0){
                System.out.println(i);
            }
        }
    }
}

File: Daily/August/06-August-2025/3.cpp
// Online C++ compiler to run C++ program online
#include <iostream>
#include <math.h>
using namespace std;

int binaryToDecimal(int number) {
    int decimal=0;
    int i=0;
    while (number) {
        int bit= number%10;
        decimal= decimal+bit*pow(2, i++);
        number = number/10;
    }
    return decimal;
}

int main() {
    int number;
    cout << "Enter binary number: ";
    cin>>number;
    cout << "Decimal number: " << binaryToDecimal(number)<<endl;
    
    return 0;
}